src/pages/tools.astro

```diff
@@ -0,0 +1,39 @@
+---
+import i18next, { t, changeLanguage } from "i18next";
+import Base from "../layouts/Base.astro";
+
+changeLanguage("en");
+
+const title = `Publisher tools | ${t("site.title")}`;
+const description = t("site.description");
+---
+<Base title={title} description={description}>
+  <section class="section section_adjusted">
+    <iframe
+      class="iframe_content"
+      src="https://wmtools.interledger-test.dev?contentOnly"
+      title="Publisher tools"
+      loading="eager"
+    />
+  </section>
+</Base>
+
+<style>
+  body {
+    background-image: url('/img/bg-tile.svg');
```

Comment:
Should either remove this background, or add a `#ffffff` on iframe, otherwise there's 2x  2x web/mash background.